shell/common/gin_converters/file_path_converter.h

```diff
@@ -22,6 +21,12 @@ struct Converter<base::FilePath> {
     if (val->IsNull())
       return true;
 
+    v8::String::Value str(isolate, val);
+    if (str.length() == 0) {
```

Comment:
Noted in DM but we're gonna try upstream this to Chromium's new `base::string16` converter as it's a flaw in their implementation